packages/geoip-location/test/unit/tarHelper.test.ts

```diff
@@ -0,0 +1,102 @@
+import { extractFileFromTarStream } from '../../src/tarHelper'
+import { TestServer } from '../helpers/TestServer'
+
+let sPort = 3197
+
+const getServerPort = () => {
+    const port = sPort
+    sPort++
+    return port
+}
+describe('tarHelper', () => {
```

Comment:
Start and stop server is beforeEach and afterEach

Reply:
Putting server start and stop in tests themselves is intentional. The servers are unique, they run on different ports and some of them get aborted in the middle. I tried using beforeEach() and afterEach(), which resulted in the servers of wrong test tests cases getting closed.